### ISSUE TITLE
Talkback doesn't read the first letter of text in charactor granularity reading mode.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#218]
```
What steps will reproduce the problem?
1. open any text with Talkback on
2. change the reading granularity to "characters".
3. swipe to find the first letter of the block of text.

What is the expected output?
Talkback should read all chars.

What do you see instead?
The first char is replaced by a sort of ...donk... sound.


What version of the product are you using?
Talkback version 3.3.1

On what operating system?
Android 4.2.2 [jellybean]


Please provide any additional information below.
This seems to happen irregardless of which application the phone is in, or 
where the Talkback focus is.

```

Original issue reported on code.google.com by `thomasc...@gmail.com` on 11 Apr 2013 at 12:18

**Comment by GoogleCodeExporter** [comment]
```
This is a result of a bug in the accessibility framework for character 
navigation.  We've fixed the issue, but it will require an Android OTA cycle to 
reach devices.
```

Original comment by `caseybur...@google.com` on 12 Apr 2013 at 3:31
- Changed state: **Fixed**
